src/components/NewClusterModal.js

```diff
@@ -141,6 +141,8 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
     const currentPersistentDisk = this.getCurrentPersistentDisk()
 
     this.state = {
+      // Uncomment this to enable the modal state debugger pane. Leave undefined for production.
+      // showDebugger: false,
```

Comment:
TOL: I don't personally tend to like undefined as a trick for this, because it doesn't document itself as well in my opinion. If we want to start using this debug mode as a pattern, I might rather have a `debugModeEnabled` flag in the config (or something) so we can enable debug mode for local without enabling for prod. I think that also reduces the risk of the debug code accidentally hitting production by mistake.

Reply:
Yeah, I was definitely taking the shortcut of using `undefined` as a special value instead of adding another distinct piece of state. Seems like there's a bit more to explore here to decide how we want to organize our debugging utilities.

Reply:
I am okay with doing this as a temporary step, but would want there to be a plan for moving to the new method of state. Do you have thoughts on how you would want to move forward on that after this PR/feature is released?